error-analyzer/src/main/java/com/google/sps/servlets/helloInLog4j2.java

```diff
@@ -0,0 +1,27 @@
+package com.google.sps.servlets;
```

Comment:
Have we decided upon a directory structure? If so, can you please explain it to me.